core/domain/exp_services.py

```diff
@@ -1757,3 +1757,34 @@ def get_interaction_id_for_state(exp_id, state_name):
         return exploration.get_interaction_id_by_state_name(state_name)
     raise Exception(
         'There exist no state in the exploration with the given state name.')
+
+
+def save_multi_exploration_math_rich_text_info_model(
+        multiple_explorations_math_rich_text_info):
+    """Saves multiple instances of ExplorationMathRichTextInfoModel to the
+    datastore.
+
+    Args:
+        multiple_explorations_math_rich_text_info:
+        tuple(ExplorationMathRichTextInfoModel, str). list of
+        ExplorationMathRichTextInfoModel domain object and corresponding
+        exploration ID.
+    """
+
+    exploration_math_rich_text_info_models = []
+    for exploration_math_rich_text_info, exp_id in (
+            multiple_explorations_math_rich_text_info):
+        latex_values_without_svgs = (
+            exploration_math_rich_text_info.latex_values_without_svgs)
+        estimated_max_size_of_images_in_bytes = (
+            exploration_math_rich_text_info.get_svg_size_in_bytes())
+        exploration_math_rich_text_info_models.append(
+            exp_models.ExplorationMathRichTextInfoModel(
+                id=exp_id,
+                math_images_generation_required=True,
```

Comment:
Add both of these values (id, id) in the domain object! (This is a save method it should not save random values in the models.)

Reply:
done